conanfile.py

```diff
@@ -218,6 +218,16 @@ def _get_source(self):
         tools.get(url)
         os.rename(f"cpython-{self.conanfile.pyversion}", "src")
 
+        # Patch a build issues with Clang 13: https://bugs.python.org/issue45405
+        compiler = self.conanfile.settings.compiler
+        if compiler == "clang" and tools.Version(self.conanfile.pyversion) < "3.9.8":
```

Comment:
Maybe worth adding to the comment here that we apply to all `clang` versions as previously the `--print-multiarch` flag did nothing anyway?

Reply:
Good point! I added the comment.